shared/js/background/utils.es6.js

```diff
@@ -306,6 +306,23 @@ function isSameTopLevelDomain (url1, url2) {
     return firstDomain === secondDomain
 }
 
+/**
+ * Checks the config to see if a feature is enabled. You can optionally pass a second "customState"
+ * parameter to check if the state is equeal to other states (i.e. state === 'beta').
+ *
+ * @param {String} featureName - the name of the feature
+ * @param {String} customState - An optional custom state to check for
+ * @returns {bool} - if feature is enabled/matches specified state
+ */
+function isFeatureEnabled (featureName, customState) {
+    const feature = tdsStorage.config.features[featureName]
+    if (!feature) {
+        return false
+    }
```

Comment:
The default state (when the config for the feature is missing) is disabled? This means that if a feature name changed (e.g. due to refactoring), all old versions of the extension would disable that feature. I think a case for both defaults could be made, but maybe worth enumerating pros and cons.

Reply:
We have a default configuration embedded into the extension, maybe we should merge it with the remote one (prioritizing the remote one)?

Reply:
So are we saying when we get the remote config to merge it in to the default config which would:
- Overwrite existing keys
- Keep keys in the default config that don't exist in the remote config?